doc/distrib/NodeHelpFiles/_review/DSRevitNodesUI.ElementsInView.md

```diff
@@ -0,0 +1,10 @@
+## In Depth
+`All Elements In Active View` returns all of the elements that are visible in the active view of the current Revit document.
+
+`All Elements In Active View` will update on as the view changes in Revit, resulting in a new list.
```

Comment:
- The word "on" before "as the view changes" is unnecessary.
- "update as the view changes in Revit" might create confusion as it won’t update unless the graph is run again if in manual mode. Suggest adding something along the lines of "the view changes in Revit and the dynamo graph is run" for clarity.